doc/source/guide/data_types/maps.rst

```diff
@@ -117,20 +117,20 @@ Basic statistical functions on the data array are also passed through to Map obj
 
 5. Creating a plot of your map
 ------------------------------
-The SunPy map object has its own built-in plot methods so that it is easy to
+The SunPy `~sunpy.map.GenericMap` object has its own built-in plot methods so that it is easy to
 quickly view your map on the screen. To create a plot just type::
 
     my_map.peek()
-    
+
 This will open a matplotlib plot right on your screen.
 In addition, to enable users to modify the plot it is possible to grab the
-matplotlib figure object by using the plot() command.
-This makes it possible to use the SunPy plot as the foundation for a 
+matplotlib figure object by using the `~sunpy.map.GenericMap.plot()` command.
+This makes it possible to use the SunPy plot as the foundation for a
 more complicated figure.
 
 .. note::
-    
-   If the `wcsaxes <http://wcsaxes.readthedocs.org/en/latest/>`_ package is not 
+
+   If the `wcsaxes <http://wcsaxes.readthedocs.org/en/latest/>`_ package is not
```

Comment:
WCSAxes is not listed as an optional package for sunpy in the installation guide.  The installation guide should also give a short description as to what WCSAxes does.